### PR TITLE
PLANET-7462: Remove Lodash dependency from enforms.js

### PR DESCRIPTION
**Pull request by lithrel**
Ref: https://jira.greenpeace.org/browse/PLANET-7462
Following: https://jira.greenpeace.org/browse/PLANET-7323
Following: https://jira.greenpeace.org/browse/PLANET-7304

Requires: https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1197

- Replace `_.template` functionality with js template literals.
- Fix missing API endpoint
- Fix side style background image path

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [neptune](https://www-dev.greenpeace.org/test-neptune) | [admin](https://www-dev.greenpeace.org/test-neptune/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-neptune/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-neptune) | [composer-local.json](https://github.com/greenpeace/planet4-test-neptune/blob/main/composer-local.json)

:watch: 2024.04.08 09:10:57

File: admin/js/enforms.js [hunk 1]
```diff
@@ -1,5 +1,185 @@
 /* global ajaxurl, Backbone, _ */
 
+const enformTemplates = {
+  enSelectedField: data => {
+    const __ = wp.i18n.__;
```
Comment (mleray) [inline admin/js/enforms.js:5]: Can't we declare this only once at the top of the page?

File: admin/js/enforms.js [hunk 2]
```diff
@@ -52,7 +232,8 @@ const p4_enform = ($ => {
    */
   app.Views.FieldsListView = Backbone.View.extend({
     el: '#en_form_selected_fields_table',
-    template: _.template($('#tmpl-en-selected-fields').html()),
+    //template: _.template($('#tmpl-en-selected-fields').html()),
```
Comment (mleray) [inline admin/js/enforms.js:235]: Is this commented line intended?

File: admin/js/enforms.js (hunk 1 above)
Comment (lithrel) [reply admin/js/enforms.js:5]: I'm a tiny bit afraid to declare things in global context :grimacing:

**Pull request merged**